libethashseal/genesis/eip158Test.cpp

```diff
@@ -39,7 +39,8 @@ R"E(
 		"blockReward": "0x4563918244F40000",
 		"registrar" : "0xc6d9d2cd449a754c494264e1809c50e34d64562b",
 		"networkID" : "0x1",
-		"chainID": "0x01"
+		"chainID": "0x01",
+		"allowFutureBlocks" : "1"
```

Comment:
So we have values like `0x1`, `0x01` and `1`. WTF? Why it cannot be `true` / `false`?